src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -2,20 +2,46 @@
 
 import java.sql.Connection;
 import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 public class DatabaseManager {
     private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
 
     private Connection connection;
 
-    boolean connect(String database, String user, String password) {
+    public boolean connect(String database, String user, String password) {
         try {
             Class.forName("org.postgresql.Driver");
-            connection = DriverManager.getConnection(DB_CONNECTION_URL + database, user, password);
+            connection = DriverManager.getConnection(DB_CONNECTION_URL + database + "?loggerLevel=OFF", user, password);
             return true;
         } catch (ClassNotFoundException | SQLException e) {
             return false;
         }
     }
+
+    public String[] getTableNames() throws SQLException {
+        String sqlQuery = "SELECT relname FROM pg_stat_user_tables";
+        try (Statement statement =connection.createStatement();
+             ResultSet resultSet = statement.executeQuery(sqlQuery)) {
+            int size = sizeOfQuery("pg_stat_user_tables");
+            String[] tableNames = new String[size];
+            int index = 0;
+            while (resultSet.next()) {
+                tableNames[index++] = resultSet.getString(1);
+            }
+            return tableNames;
+        }
+    }
+
+    private Integer sizeOfQuery(String tableNameWhereQuery) throws SQLException {
```

Comment:
"tableNameWhereQuery" may be rename it?) I can't understand what does it mean.

Reply:
I guess you're right... And the name of method is also not good. A new day a new vision ))) Thanks